src/Microsoft.Azure.WebJobs.Host/Triggers/CommonTriggerBinding.cs

```diff
@@ -7,11 +7,10 @@
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Listeners;
 using Microsoft.Azure.WebJobs.Host.Protocols;
-using Microsoft.Azure.WebJobs.Host.Triggers;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.Host.Triggers
 {
-    // Generally usable binder. 
+    // Generally usable binding for trigger parameters using the trigger hook strategy. 
     internal class CommonTriggerBinding<TMessage, TTriggerValue> : ITriggerBinding
```

Comment:
you're right - I missed this.  I renamed to StrategyTriggerBinding since it's based on the strategy object.